src/client/src/commons/form/borehole/completion/backfill.js

```diff
@@ -99,7 +99,7 @@ const Backfill = ({ isEditable, completionId }) => {
         <Stack direction="row" justifyContent="flex-end" alignItems="center">
           {isEditable && (
             <Tooltip title={t("add")}>
-              <Button
+              <BdmsButton
                 data-cy="add-backfill-button"
                 variant="outlined"
                 startIcon={<AddIcon />}
```

Comment:
PP: Diese beiden Properties auch gleich in die Komponente schieben, da wir zur Zeit immer die gleichen Werte dafür definieren. Falls es mal anders sein sollte, kann man diese dann von _aussen_ konfigurierbar machen.